### DISCUSSION TITLE
Why firstname and lastname fields are not available in Graphql?

### DISCUSSION THREAD

**Discussion by jahidanowar** [strapi/strapi#8136] [Legacy Topics]
I am creating a blog so I have created a Post content type when I am viewing the Rest API I can see the created_by and updated_by fields with Creator First name and last name. But when I am trying to fetch the same using graphql it's not working. 

Here is my Graphql Query 

```gql
query Posts {
  posts {
    id
    updated_at
    slug
    Title
    content
    Image {
      url
    }
    created_by{
      firstname
      lastname
    }
    categories {
      id
      Name
    }
    comments {
      Name
      Content
    }
  }
}

```

**Comment by derrickmehaffy** [comment]
Hi there :)

Just a heads up, we will be getting rid of GitHub Discussions sometime in Q1 2021, in the future can you please ask your question on our Forum.

The `created_by` and `updated_by` weren't meant to be added into the schema and are due to be removed (they are used for the Admin Panel), see the following PR: https://github.com/strapi/strapi/pull/8052

**Reply by jahidanowar** [reply]
```json
  "options": {
    "increments": true,
    "timestamps": true,
    "privateAttributes": ["id", "created_at"],
    "populateCreatorFields": true
  },
```

I am still not getting the creator firstname or lastname using GraphQL

**Comment by jahidanowar** [comment]
```json
  "options": {
    "increments": true,
    "timestamps": true,
    "privateAttributes": ["id", "created_at"],
    "populateCreatorFields": true
  },
```
I am still not getting the creator firstname or lastname using GraphQL

**Reply by derrickmehaffy** [reply]
Can you open up a bug report for that?
